### PR TITLE
Site: add a note for default profile name in Docker Desktop Replacement doc

### PR DESCRIPTION
**Pull request by klaases**
Fixes: #14946

Per request, added a note to indicate the default profile name, which will help first-time users of minikube.

**Before:**
Screen Shot 2022-10-11 at 16 54 45

**After:**
Screen Shot 2022-10-12 at 17 34 13

### CODE REVIEW

File: site/content/en/docs/tutorials/docker_desktop_replacement.md [hunk 1]
```diff
@@ -87,6 +87,8 @@ minikube start --container-runtime=docker --vm=true
 <h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">3</strong></span>Point Docker CLI to minikube</h2>
 Use the `minikube docker-env` command to point your terminal's Docker CLI to the Docker instance inside minikube.
 
+<br>Note: the default minikube profile name is "default".
```
Comment (spowelljr) [inline site/content/en/docs/tutorials/docker_desktop_replacement.md:90]: The default minikube profile is `minikube`

**Review by spowelljr — CHANGES_REQUESTED** [review]

File: site/content/en/docs/tutorials/docker_desktop_replacement.md (hunk 1 above)
Comment (klaases) [reply site/content/en/docs/tutorials/docker_desktop_replacement.md]: Was moving too fast, thanks for the catch!

File: site/content/en/docs/tutorials/docker_desktop_replacement.md [hunk 2]
```diff
@@ -87,6 +87,8 @@ minikube start --container-runtime=docker --vm=true
 <h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">3</strong></span>Point Docker CLI to minikube</h2>
 Use the `minikube docker-env` command to point your terminal's Docker CLI to the Docker instance inside minikube.
 
+<br>Note: the default minikube profile name is "minikube".
```
Comment (medyagh) [inline site/content/en/docs/tutorials/docker_desktop_replacement.md:90]: <br>Note: if you are using a different profile name than default (minikube) you can pass it by -p

Comment (medyagh) [reply site/content/en/docs/tutorials/docker_desktop_replacement.md:90]: and then remove all the "-p <profiles>" in the instructions, so the users could just copy/paste

**Review by medyagh — CHANGES_REQUESTED** [review]

File: site/content/en/docs/tutorials/docker_desktop_replacement.md (hunk 2 above)
Comment (klaases) [reply site/content/en/docs/tutorials/docker_desktop_replacement.md]: Yes, this makes sense, applied suggestions.

**Pull request closed without merging**